### ISSUE TITLE
GB: affiliation with parliament (lower and upper)

### ISSUE THREAD

**Issue by matyaskopp** [clarin-eric/ParlaMint#219]
Following example has several issues:
```XML
               <person xml:id="SayeedaWarsi">
                  <persName>
                     <forename>Sayeeda</forename>
                     <surname>Warsi</surname>
                  </persName>
                  <sex value="F">Female</sex>
                  <affiliation from="2007-10-11" ref="#parla.lower" role="MP"/>
                  <affiliation from="2007-10-11" ref="#party.CON" role="member"/>
                  <affiliation from="2010-05-12" ref="#PoGB" role="minister" to="2012-09-06"/>
                  <affiliation from="2012-09-06" ref="#PoGB" role="minister" to="2014-08-05"/>
                  <idno subtype="contact" type="URI">https://members.parliament.uk/member/3839/contact</idno>
                  <figure>
                     <graphic url="https://api.parliament.uk/photo/Paa3j0vS.jpg?crop=CU_1:1"/>
                  </figure>
               </person>
```

1. affiliation has to be with the organization.  `parla.lower` refers to a category
2. Sayeeda Warsi is not a member of the lower house. She is a member of the lords (I am suggesting using `role="MP"` - I know that it is not ideal in the UK's political system, but we use `MP` for both chambers)
3. I believe that a minister should be a member of a government, not a member of parliament (see discussion #185 )

Regarding 1 and 2: Is it possible to distinguish two organizations (**house of commons** and **house of lords**) and their terms stored in `<event>` element?

@perayson, @matthewcoole - Does this make sense in the UK's universe?

**Comment by matyaskopp** [comment]
> I think perhaps for the UK the only way to really resolve the role issue would be to introduce a new `role="peer"`.

We have an MP role for the person who votes on laws and is a part of the legislative procedure. I don't think we want to introduce a new role.
@TomazErjavec  adopted a British naming "MP" for deputies but using the `peer` role for upper house members is odd for other parliaments.
@katjameden, @coltekin What is your opinion?

> 3. This is an interesting take. However a minister is still an MP whilst acting in their role as a minister (they can even have multiple ministerial roles at once).

I can say it differently: A person who has a minister role in government can also have an MP role in parliament. So there are two affiliations. 
We have to discuss how to implement a ministry - a person can be one more affiliation with ministry (or multiple ministries...)

**Comment by TomazErjavec** [comment]
What about having the roles for the upper and lower house the the same, and both being simply "member"?

**Comment by matyaskopp** [comment]
related discussion: #16

> What about having the roles for the upper and lower house the the same, and both being simply "member"?

If I correctly understand, you suggest not using the `MP` role at all.

I have some observations on that:
- it brakes CZ's` candidateMP` role. Changing it to `candidate` doesn't make sense because it describes a relationship with a political party, and it should be clear that a person is a candidate for parliament (not to a political party)
- imagine a situation where someone wants to list all persons with "voting right in parliament" = members of organizations with `@ana="parla.national"` or `@ana="parla.regional"`. It is too complicated.

I think we should preserve a unique role for parliament membership, but I am not saying it should be `MP`, which is confusing.

What about the `legislator` role?

**Comment by TomazErjavec** [comment]
> > What about having the roles for the upper and lower house the the same, and both being simply "member"?
> 
> If I correctly understand, you suggest not using the `MP` role at all.

Yes, exactly. On the plus side, it would then be the same as e.g. for party (and other) memberships, so nicely consistent  regardless of the organisation they are a member of. Also, we would get rid of the country-based naming of these roles in the XML. And, thinking about what "MP" means, we are now saying somebody is a "Member of Parliament of Parliament", rather redundant.

However, when generating the vertical file for the concordancer or the metadata TSV, we would now have to resolve "member" of the parliament to speaker_type="MP" (or similar, like speaker_type="legislator" ), which would be more complicated than it is now. Now we just check that the role is "MP", but afterwards we would have to check that they are a member of an organisation that is the parliament. Yes, more, complicated, but we do the same thing party membership already, so maybe the same code could be reused.

> I have some observations on that:
> 
>     * it breaks CZ's` candidateMP` role. Changing it to `candidate` doesn't make sense because it describes a relationship with a political party, and it should be clear that a person is a candidate for parliament (not to a political party)

If we were to change to "member" then you would have to change to "candidateMember" or something similar, I don't see that is a big problem. Note that this is always an affiliation, so you know that the "candidateMember" refers to the parliament and means "candidateMP".

>     * imagine a situation where someone wants to list all persons with "voting right in parliament" = members of organizations with `@ana="parla.national"` or `@ana="parla.regional"`. It is too complicated.

Well, yes, it is complicated in XML, but, as above, it is already complicated for political parties, esp. as you also need to take into account the time-frame when speaking. But the compications are in the conversion scripts, so in the TSV or vertical files, it won't be any more complicated than now.

> I think we should preserve a unique role for parliament membership, but I am not saying it should be `MP`, which is confusing. What about the `legislator` role?

If you are still not convinced about the simpicity (at least in representation) of "member", I guess it could be "legislator". But (and more about this separately) we really should consult some poitical scientist about this terminilogy.

**Comment by matyaskopp** [comment]
> If we were to change to "member" then you would have to change to "candidateMember" or something similar, I don't see that is a big problem. Note that this is always an affiliation, so you know that the "candidateMember" refers to the parliament and means "candidateMP".

thinking about it "electionCandidate" is quite a good option.

> If you are still not convinced about the simpicity (at least in representation) of "member", I guess it could be "legislator". But (and more about this separately) we really should consult some poitical scientist about this terminology.

ok, opening a new issue for issues that need to be discussed with an expert: #220 
I have to give it a few days - maybe I will accept "member"

**Comment by TomazErjavec** [comment]
It seems "member" is accepted, further discussion in #237.
